src/replays/_replayswidget.py

```diff
@@ -501,19 +501,24 @@ def onlineTreeClicked(self, item):
                     item.generateInfoPlayersHtml()
 
     def onlineTreeDoubleClicked(self, item):
-        if hasattr(item, "duration"):
+        if hasattr(item, "duration"):  # it's a game not a date separator
             if "playing" in item.duration:  # live game will not be in vault
-                if not item.live_delay:  # live game under 5min
-                    name = None
-                    if item.mod == "ladder1v1":
-                        name = item.name[:item.name.find(" ")]  # "name vs name"
-                    else:
-                        players = (p for t in item.teams for p in t)
-                        for player in players:
-                            name = player["name"]
-                            if name != "" and name in self._playerset:
+                # search result isn't updated automatically - so game status might have changed
+                if item.uid in client.instance.gameset.games:  # game still running
+                    game = client.instance.gameset.games[item.uid]
+                    if not game.launched_at:  # we frown upon those
+                        return
+                    if game.has_live_replay:  # live game over 5min
```

Comment:
I often find myself double-clicking the 'crossed swords' icon in chat before the 5 minute period ends and just waiting for the game to start playing the replay. I think it might be a use case here too. I don't really know what would be the best way to handle it, maybe some popup? What do you think?

Reply:
yeah though about that .. a: 'too early .. wait n:nn to watch the live game'
(but I always hesitate to complicate things .. cause then you brakedown and cry ;-)

Reply:
Well, an alternative to a popup is an angry user not knowing why the game doesn't launch even though he double-clicked. Alternatively we can just proceed with launch anyway, people might be already familiar with how launching delayed games works.

Reply:
I'll make a msg.